app/styles/lists.styl

```diff
@@ -15,7 +15,7 @@ $item-margin = .7em
     position         relative
     border-radius    4px
     background-color white
-    box-shadow       0 2px 6px 0 alpha(black, .15)
+    box-shadow       0 0 1px 0 #000
```

Comment:
I'd rather not see any HEX color in CSS. To be consistent you know. Even with `box-shadow`, you should use variables or, in this case, CSS keywords like `black` is just fine.